float64 a_traj
